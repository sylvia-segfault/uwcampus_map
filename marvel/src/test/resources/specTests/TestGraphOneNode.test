# Test the parser on a graph that contains one node

LoadGraph oneNodeGraph oneNode.tsv
ListNodes oneNodeGraph
ListChildren oneNodeGraph THING/BENJAMIN_J._GR