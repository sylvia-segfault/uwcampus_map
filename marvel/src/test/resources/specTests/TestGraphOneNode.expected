# Test the parser on a graph that contains one node

loaded graph oneNodeGraph
oneNodeGraph contains: THING/BENJAMIN J. GR
the children of THING/BENJAMIN J. GR in oneNodeGraph are: